src/java/fr/paris/lutece/plugins/workflow/modules/formspdf/service/HtmlToPDFGenerator.java

```diff
@@ -195,23 +283,185 @@ private void writeExportFile( Path directoryFile ) throws IOException
         {
             strError = "Une erreur s'est produite lors de la generation de l'edition";
             AppLogService.error( strError, e );
-            throw new RuntimeException( strError, e );
+           throw new RuntimeException( strError, e );
         }
         catch( IOException e )
         {
             strError = "Une erreur s'est produite lors de la generation de l'edition";
             AppLogService.error( strError, e );
-            throw new RuntimeException( strError, e );
+           throw new RuntimeException( strError, e );
         }
 
     }
-    
     private void markersToModel( Map<String, Object> model, Collection<InfoMarker> collectionInfoMarkers )
     {
         for ( InfoMarker infoMarker : collectionInfoMarkers )
         {
-            model.put( infoMarker.getMarker(), infoMarker.getValue() );           
+            model.put( infoMarker.getMarker(), infoMarker.getValue() );
         }
     }
 
+    private Map<String, Object> markersToModel( Map<String, Object> model, Collection<InfoMarker> collectionInfoMarkers, List<FormQuestionResponse> formQuestionResponseList )
+    {
+        HashMap<Integer, FormQuestionResponse> formResponseListByEntryId = formResponseListToHashmap(formQuestionResponseList);
+        String adminBaseUrl = "";
+        if(StringUtils.isBlank(AppPropertiesService.getProperty( "lutece.admin.prod.url" )))
```

Comment:
Please use AppPathService.getProdUrl( _baseUrl)

Reply:
ok, i changed this part by  adminBaseUrl = _baseUrl;
because _baseUrl was already set by AppPathService.getProdUrl(  request ); in src/java/fr/paris/lutece/plugins/workflow/modules/formspdf/service/task/FormsPDFTask.java